datasource/http/data_acc_test.go

```diff
@@ -57,7 +57,7 @@ func TestHttpDataSource(t *testing.T) {
 			Path:  testDatasourceInvalidMethod,
 			Error: true,
 			Outputs: map[string]string{
-				"error": "the `method` must be one of [HEAD GET POST PUT DELETE OPTIONS PATCH]",
+				"error": "the `method` must be one of \\[HEAD GET POST PUT DELETE OPTIONS PATCH\\]",
```

Comment:
nit: I would rename Outputs to ExpectedOutputs, I know its unrelated to this PR, and feel free to reject it, but it was unclear to me on first glance that this value is doing a direct assertion against the output.

Reply:
It is indeed unrelated to this PR, but I do agree with you that it'd be clearer.
I'll push that as a separate commit in this PR.